python_modules/libraries/dagster-aws/dagster_aws/pipes/clients/emr_serverless.py

```diff
@@ -318,6 +325,21 @@ def _read_messages(
                     ),
                 )
 
+    def _extract_dagster_metadata(self, response: "GetJobRunResponseTypeDef") -> RawMetadataMapping:
+        metadata: RawMetadataMapping = {}
+
+        job_run = response["jobRun"]
+
+        metadata["AWS EMR Serverless Application ID"] = job_run["applicationId"]
+        metadata["AWS EMR Serverless Job Run ID"] = job_run["jobRunId"]
```

Comment:
Is the particular form of these metadata keys just kind of made up or does it match something somewhere? I could be wrong here but I don't think I've seen us using metadata keys with spaces elsewhere. If this is an arbitrary decision would recommend `snake_case`.